keel-tagging-plugin/src/main/kotlin/com/netflix/spinnaker/keel/tagging/ResourceTagger.kt

```diff
@@ -200,14 +195,20 @@ class ResourceTagger(
       account
     }
 
+    val (region, resourceId) = if (suffix.contains(":")) {
+      suffix.split(":", limit = 2).toList()
+    } else {
+      listOf("*", suffix)
+    }
```

Comment:
Cluster resource ids no longer include a region. In the entity tag that should be reflected as `*` for the region field.